Documentation/README.md

```diff
@@ -17,9 +17,11 @@ We’ve designed the system to make it really easy to share packages on services
     * [Dependency Hell](#dependency-hell)
```

Comment:
Maybe we could also link to this `Documentation/README.md` from the repository-level `README.md`?

Reply:
Good point, I think right now we link to the directory instead